lib/ws_servers/api/session.js

```diff
@@ -115,6 +130,15 @@ class Session {
     this.credentials[mode] = { apiKey, apiSecret }
   }
 
+  sendDataToMetricsServer (data) {
+    return (data) => {
```

Comment:
this `data` args overrides the one above.

Reply:
the data arg above is not required.. forgot to remove it later. thanks!!